Dockerfile

```diff
@@ -21,6 +13,12 @@ RUN if [ "$INSTALL_CPU_PROFILER" = "true" ]; then \
         rm -rf /pyflame; \
     fi
 
+# Make a directory for private credentials files
+RUN mkdir /auth
```

Comment:
I thought we'd changed this before? Maybe I'm thinking of a different place?

Reply:
Yes, that just hadn't been merged when you reviewed this